client/me/get-apps/desktop-download-card.jsx

```diff
@@ -0,0 +1,191 @@
+				return translate( 'Requires Mac OS X 10.11+. ' );
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return translate( 'Requires a 64-bit kernel. ' );
+			default:
+				return translate( 'Requires Windows 7+. ' );
+		}
+	}
+
+	getTranslateComponents = ( platform ) => {
+		switch ( platform ) {
+			case 'MacIntel':
+				return {
+					firstAvailableLink: this.getLinkAnchorTag( this.windowsLink ),
+					secondAvailableLink: this.getLinkAnchorTag( this.linuxTarLink ),
+					thirdAvailableLink: this.getLinkAnchorTag( this.linuxDebLink ),
+				};
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return {
+					firstAvailableLink: this.getLinkAnchorTag( this.linuxDebLink ),
+					secondAvailableLink: this.getLinkAnchorTag( this.windowsLink ),
+					thirdAvailableLink: this.getLinkAnchorTag( this.macLink ),
+				};
+			default:
+				return {
+					firstAvailableLink: this.getLinkAnchorTag( this.macLink ),
+					secondAvailableLink: this.getLinkAnchorTag( this.linuxTarLink ),
+					thirdAvailableLink: this.getLinkAnchorTag( this.linuxDebLink ),
+				};
+		}
+	}
+
+	getAlsoAvailableText = ( platform ) => {
+		const { translate } = this.props;
+		switch ( platform ) {
+			case 'MacIntel':
+				return translate( 'Also available for: ' +
+					'{{firstAvailableLink}}Windows{{/firstAvailableLink}}, ' +
+					'{{secondAvailableLink}}Linux (.tar.gz){{/secondAvailableLink}}, ' +
+					'{{thirdAvailableLink}}Linux (.deb){{/thirdAvailableLink}}.',
+					{ components: this.getTranslateComponents( platform ) }
+				);
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return translate( 'Also available for: ' +
+					'{{firstAvailableLink}}Linux (.deb){{/firstAvailableLink}}, ' +
+					'{{secondAvailableLink}}Windows{{/secondAvailableLink}}, ' +
+					'{{thirdAvailableLink}}Mac{{/thirdAvailableLink}}.',
+					{ components: this.getTranslateComponents( platform ) }
+				);
+			default:
+				return translate( 'Also available for: ' +
+					'{{firstAvailableLink}}MacOS{{/firstAvailableLink}}, ' +
+					'{{secondAvailableLink}}Linux (.tar.gz){{/secondAvailableLink}}, ' +
+					'{{thirdAvailableLink}}Linux (.deb){{/thirdAvailableLink}}.',
+					{ components: this.getTranslateComponents( platform ) }
+				);
+		}
+	}
+
+	getButtonClickHandler = ( platform ) => {
+		switch ( platform ) {
+			case 'MacIntel':
+				return this.trackMacClick;
+			case 'Linux i686':
+				return this.trackLinuxTarClick;
+			case 'Linux i686 on x86_64':
+				return this.trackLinuxDebClick;
+			default:
+				return this.trackWindowsClick;
+		}
+	}
+
+	getButtonLink = ( platform ) => {
+		switch ( platform ) {
+			case 'MacIntel':
+				return this.macLink;
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return this.linuxTarLink;
+			default:
+				return this.windowsLink;
+		}
+	}
+
+	getCardTitle = ( platform ) => {
+		const { translate } = this.props;
+		switch ( platform ) {
+			case 'MacIntel':
+				return translate( 'Desktop App for Mac' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Desktop App' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).